encoding/resampling.md

```diff
@@ -374,9 +381,9 @@ so as to not introduce banding.
 Example code for resizing in linear RGB light:
 
 ```py
-linear = core.resize.Bicubic(src, format=vs.RGBS, transfer_in_s="709", transfer_s="linear", matrix_in_s="709")
+linear = core.resize.Bicubic(src, format=vs.RGBS, transfer_in_s="709", transfer_s="linear", matrix_in_s="709") #  the matrix_in_s argument is only necessary when src is YUV; otherwise, it should be omitted
```

Comment:
Consiering the length of the line, this comment should be placed in a line before the code. Same below.